homeassistant/components/netgear/config_flow.py

```diff
@@ -124,16 +124,16 @@ async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowRes
         """Initialize flow from ssdp."""
         updated_data: dict[str, str | int | bool] = {}
 
+        _LOGGER.debug("Netgear ssdp discovery info: %s", discovery_info)
+
+        await self.async_set_unique_id(discovery_info.upnp[ssdp.ATTR_UPNP_SERIAL])
+        self._abort_if_unique_id_configured()
+
         device_url = urlparse(discovery_info.ssdp_location)
         if hostname := device_url.hostname:
```

Comment:
Keep the updating, but only pass the IP update if it's an IPv4. We have `is_ipv4_address` helper in `util.network`